packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -117,38 +132,30 @@ export class ColumnHeaderCell extends AbstractPureComponent<ColumnHeaderCellProp
         );
     }
 
-    public context: ColumnInteractionBarContextTypes = {
```

Comment:
turns out there is a small functional change in ColumnHeaderCell2 having to do with usage of legacy React context. this feature was reimplemented in Table2 by injecting props. I've now ported that implementation to Table as well, so we no longer need legacy context :tada: